src/dot/reseedDots.m

```diff
@@ -35,7 +35,7 @@
 
         dots.positions(N, :) = positions;
         dots.speeds(N, :) = speeds;
-        dots.time(N, 1) = time;
```

Comment:
Makes sense to me to do this indeed.

But now that we don't use the `time` variable, you can remove it from line 34 and only keep:

```
[positions, speeds] = seedDots(dots, cfg, isSignal);
```

Reply:
So, now line 34 is clean. Though I am not confident to delete the `time` calculation in `seedDots`. I am afraid to break things if this is used in another part. (We do not have a dependency visualizer tool yet, do we?)

https://github.com/cpp-lln-lab/CPP_PTB/blob/192b180ad6483adb9f070225dcee9693de342a98/src/dot/seedDots.m#L25-L28